index.js

```diff
@@ -1,4 +1,4 @@
-const request = require('@derhuerst/gemini/client')
+const request = require('@bwestergard/gemini/client')
```

Comment:
Would it be possible to get everything merged into the original repo instead of needing a fork?

https://github.com/derhuerst/gemini